visualize/grpc/pb/visualize_grpc.pb.go

```diff
@@ -1883,6 +1883,100 @@ func (x *SetNodeModeEvent) GetNodeMode() *NodeMode {
 	return nil
 }
 
+type CommandRequest struct {
```

Comment:
Is this file automatically generated by protobuf/gRPC compiler? Should we remove auto-gen files and let them be created at build-time?

Reply:
Agreed.

Reply:
Yes it is. 

I perfer (strongly) that these generated files should be included in the source code. 
The benefit is that users don't need to install the tools for compiling protobuf and packing web related files, including:

- protoc
- protoc-gen-go
- protoc-gen-grpc-web
- nodejs
- go-bindata

Currently the only required software for end-user other than `otns` itself is the `grpcwebproxy`. I believe it is important to have a better user experience that users are able to install `OTNS` easier and more quickly. 

Why do end-users who just want to run otns needs to install nodejs anyway? 

Thoughts ? @wgtdkp

Reply:
Another related aspect is that many softwares written in Golang can be installed simply by executing `go get github.com/user/name` without downloading it.

`OTNS` had tried to achieve this too (i.e. installed easily by `go get github.com/openthread/ot-ns/cmd/...`), but it didn't work out because `grpcwebproxy` needs to be installed otherway. But still, it is an objective that OTNS is still being driven towards.

Reply:
In Silk repo we chose not to include the generated python proto files and add the dependencies and generation steps into the bootstrap script. I do prefer generated files to be left out since I feel they aren't really meaningful for code review and adds clutter to it. But relieving user of the generation step is also a valid point, and it's just a balancing act between user experience and code review experience. If you feel strongly about it, I'm okay with it.

Reply:
Silk only uses the proto to python generator. Even so, I have ran into cases where I can't build the required files and had to search for how on Google. 

But in OTNS, we need to use many tools more as I have listed. 

For code review, we can simply skip these files. Most of time, github would already hide these files for us.

Reply:
fine to keep it.